pkg/version/release.go

```diff
@@ -3,7 +3,7 @@ package version
 // This file was generated by release_generate.go; DO NOT EDIT.
 
 // Version is the version number in semver format X.Y.Z
-var Version = "0.1.0"
+var Version = "0.2.1"
```

Comment:
I'm confused, this should be `0.2.0` right? since its the latest version we cut

Reply:
i thought this would go in next release since we already release 0.2.0? 🤔

Reply:
sorry I'm super tired and confused :smile: if you wanna cut a `0.2.1` PR after this is merged, then yeah this makes sense to keep in this PR :D but you also need to add a `0.2.1` release notes file too

Reply:
sorry I am typing in too many threads 🙈  switched it back to `0.2.0`